tests/Unit/Domain/Creators/Test_CylindricalBinaryCompactObject.cpp

```diff
@@ -361,7 +348,16 @@ void test_construction(const std::unique_ptr<DomainCreator<3>>& creator,
   }
 }
 
-auto construct_expected_functions_of_time(const bool with_control_systems) {
+auto construct_creator_with_time_dependent_quantities(
```

Comment:
(optional because this is just a test) maybe name `construct_time_dependent`? Also it would be better to return the domain creator and std::move it if you have to (I think that's why you did the return-by-reference).